api/v1alpha1/k8ssandracluster_types.go

```diff
@@ -138,6 +189,7 @@ type CassandraDatacenterTemplateSpec struct {
 }
 
 type EmbeddedObjectMeta struct {
+	// +optional
```

Comment:
in which case would this be optional?

Reply:
The DC's namespace is set to the namespace of the K8ssandraCluster.